rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -2427,7 +2427,12 @@ void main_window::CreateConnects()
 		QStringList paths;
 
 		// Only select one folder for now
-		paths << QFileDialog::getExistingDirectory(this, tr("Select a folder containing one or more games"), qstr(fs::get_config_dir()), QFileDialog::ShowDirsOnly | QFileDialog::DontResolveSymlinks);
+		QString dir = QFileDialog::getExistingDirectory(this, tr("Select a folder containing one or more games"), qstr(fs::get_config_dir()), QFileDialog::ShowDirsOnly | QFileDialog::DontResolveSymlinks);
+		if (dir.isEmpty())
+		{
+			return;
+		}
+		paths << dir;
```

Comment:
move "QStringList paths;" above this line

Reply:
done